frontend/src/container/BillingContainer/BillingUsageGraph/BillingUsageGraph.tsx

```diff
@@ -43,6 +41,21 @@ const paths = (
 	return renderer(u, seriesIdx, idx0, idx1, extendGap, buildClip);
 };
 
+const calculateStartEndTime = (
+	data: any,
+): { startTime: number; endTime: number } => {
+	const timestamps: number[] = [];
+	data?.details?.breakdown?.forEach((breakdown: any) => {
```

Comment:
calculate the max and min on the fly itself. we do not need this array. This will save the array memory.

Reply:
Done